docs/technical_reference/flowsheets/BSM2.rst

```diff
@@ -0,0 +1,52 @@
+Benchmark Simulation Model No.2
+===============================
+
+Introduction
+------------
+
+The Benchmark Simulation Model No.2 (BSM2) is an industry benchmark for modeling a full biological wastewater
+treatment plant that includes a primary clarifier, the activated sludge process, a thickener, an anaerobic digester,
+and a dewatering unit. These unit processes are driven by biological reaction models that relate soluble and particulate wastewater
+components to their respective process rate equations. While this BSM2 flowsheet can simply be used to
+simulate and run techno-economic analyses on the operation of a conventional wastewater treatment plant,
+an additional layer of utility can be derived from using BSM2 as a baseline for comparing alternative plant
+configurations to a well-established standard and/or amongst the variations themselves by adding, removing,
+or modifying unit processes using WaterTAP's flexible modeling capabilities.
+
+Implementation
+--------------
+
+Figure 1 shows the process flow diagram for BSM2 where influent wastewater is fed
+to a primary clarifier (primary treatment); the effluent is then passed to a series of activated sludge
+reactors and a secondary clarifier (secondary treatment), and finally the sludge is passed through a thickener and
+sent to the anaerobic digester. The anaerobic digester processes the sludge to produce
+a biogas stream and residual sludge stream that passes through a dewatering unit which recycles liquid to
+the headworks of the plant while sludge is released for disposal. The flowsheet relies on the following key assumptions:
+
+   * supports steady-state only
+   * property and reaction package are provided for the activated sludge model (ASM)
+   * property and reaction package are provided for the anaerobic digester model (ADM)
+   * interfaces are provided to convert between the properties of ASM and ADM
+
+.. figure:: ../../_static/flowsheets/BSM2.png
+    :width: 800
+    :align: center
+
+    Figure 1. BSM2 flowsheet
+
+Documentation for each of the unit models can be found here:
+    * `Thickener <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/thickener.html>`_
+    * `Anaerobic digester <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/anaerobic_digester.html>`_
+    * `Dewatering unit <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/dewatering_unit.html>`_
+
+Documentation for each of the property models can be found here:
+    * `ASM1 <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ASM1.html>`_
+    * `ADM1 <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ADM1.html>`_
+
```

Comment:
We might want to include a section on `Costing`, pointing to cost relationships we use in some way (whether that be manually writing relationships in table or simply pointing to other docs where we already provide said relationships).

Then we can add to Future Refinements that we plan to improve costing relationships in terms of detail, completeness, and reasonable validity.

Reply:
Alternatively, we can just do away with the Future Refinements section and just track these ourselves in Issues, as I think we may already do for some if not all of these

Reply:
Added a quick blurb in the implementation section pointing to this page: https://watertap.readthedocs.io/en/latest/technical_reference/costing/watertap_costing.html.

I feel like if we wanted to expand more on costing, the unit model documentation would be the place to do that, not in the flowsheet documentation.

Reply:
> I feel like if we wanted to expand more on costing, the unit model documentation would be the place to do that, not in the flowsheet documentation.

This was what I was implying when I said "pointing to other docs where we already provide said relationships." Since we don't have them in unit model docs yet, looks like we'll need to add those costing relationships in (subsequent PR).  Then from here, we could just include a sentence about implementation of technical performance and costing of each unit can be found in their technical references.